components/tikv_util/src/sys/thread.rs

```diff
@@ -181,7 +181,120 @@ mod imp {
     }
 }
 
-#[cfg(not(target_os = "linux"))]
+#[cfg(target_os = "macos")]
+#[allow(bad_style)]
```

Comment:
Why allow bad style?

Reply:
Want to keep the same type names as original APIs. But Clippy suggests that `type task_inspect_t` should be `type TaskInspectT`.